pageObjects/droppablePage.js

```diff
@@ -0,0 +1,128 @@
+import { expect } from "../modules/base";
+const data = JSON.parse(JSON.stringify(require("../fixtures/testData.json")));
```

Comment:
sta je ovo? zasto stringify pa parse pa ovo pa ono? @filip-n91  @eduardPinter1  dovoljno je samo da se importuje json i to je to, radi

Reply:
Done! Stavio sam kao import I funkcionise!